chains/evm/voter/voter.go

```diff
@@ -115,3 +119,35 @@ func (w *EVMVoter) VoteProposal(m *relayer.Message) error {
 		}
 	}
 }
+
+type DynamicGasPricer struct {
```

Comment:
this is separate structure repsonsible for defining gasPrice it should be located in own package like evmgaspricer.